elastic/kibana
# [Guided onboarding] Landing page updates

## Summary
Fixes https://github.com/elastic/kibana/issues/138548 

This PR updates the Guided onboarding Landing page that can be found at `/app/home#/getting_started`. The Landing page has now 4 cards: 3 solution guides and a link to Infrastructure category. The guide card displays the state of each guide with the progress bar and a button to "start/continue" the guide.

Note: The "view integrations" button links to `/app/integrations/browse/infrastructure` but the category doesn't have any integrations added to it yet. As soon as there are any integrations in that category, the link should work as expected.

### Screenshots 
#### No guides started yet
The header button is disabled, guide cards have "view guide" buttons.

<img width="1786" alt="Screenshot 2022-10-19 at 18 46 32" src="https://user-images.githubusercontent.com/6585477/196754743-0fb32db3-db97-4d99-8e5a-ad35db2472b2.png">


#### "View guide" button is clicked
The dropdown panel shows the selected guide and the user can start the first step. The header button is enabled. 

<img width="1781" alt="Screenshot 2022-10-19 at 18 46 56" src="https://user-images.githubusercontent.com/6585477/196754776-b5d7ead0-7f03-43bf-8608-f9161498da19.png">


#### A guide is in progress
The guide card displays the progress and the "continue" button.

<img width="302" alt="Screenshot 2022-10-19 at 18 47 27" src="https://user-images.githubusercontent.com/6585477/196754835-322e295c-85cf-47c7-8a0d-e1e9795136b2.png">


#### A guide has been completed
The guide card displays the completed progress bar and the "view guide" button.

<img width="308" alt="Screenshot 2022-10-19 at 18 48 27" src="https://user-images.githubusercontent.com/6585477/196754863-400ec3b3-9c44-4a43-8b53-b29ca1a86aef.png">


### How to test
1. Enable the guided onboarding plugin by adding `guidedOnboarding.ui: true` to the file `/config/kibana.dev.yml`.
2. Start ES and Kibana with examples: `yarn es snapshot` and `yarn start --run-examples`.
3. Navigate to `/app/home#/getting_started` and check the landing page when no guides have been started yet.
4. Click the card for the search guide and check that the dropdown panel is being displayed and the first step can be started.
5. Start the first step and complete it on the page `/app/guidedOnboardingExample/stepOne`.
6. Navigate back to the landing page and check that the search card now displays the progress. 
7. Complete the search guide and check that the landing page displays the search guide as completed.   

### Checklist
